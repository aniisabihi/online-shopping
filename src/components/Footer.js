import React from "react";

const Footer = () => {
  return (
    <footer id="footer" className="page-footer">
      <p>
        A simple online shopping application &ensp;- &ensp;
        <a
          href="https://www.flaticon.com/authors/freepik"
          title="freepik icons"
        >
          Icons created by Freepik - Flaticon
        </a>
      </p>
    </footer>
  );
};

export default Footer;
