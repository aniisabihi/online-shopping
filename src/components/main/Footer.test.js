import React from "react";
import { shallow } from "enzyme";
import Footer from "./Footer";

const FooterComponent = <Footer />;

let footer = shallow(FooterComponent);

/* RENDER COMPONENT */
describe("Render Footer", () => {
  it("should render the Footer Component accordingly", () => {
    expect(footer).toMatchSnapshot();
  });
});
