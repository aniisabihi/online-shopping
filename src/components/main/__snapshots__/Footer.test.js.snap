// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Footer should render the Footer Component accordingly 1`] = `ShallowWrapper {}`;
